lib/output/utils.js

```diff
@@ -5,9 +5,9 @@ function isObject(value) {
   return typeof value === "object" && value !== null || typeof value === "function";
 }
 
-function hasOwn(obj, prop) {
-  return Object.prototype.hasOwnProperty.call(obj, prop);
-}
+/** @type {function(*, string | number | symbol): boolean} */
+const hasOwn = Function.prototype.bind.call(Function.prototype.call, Object.prototype.hasOwnProperty);
```

Comment:
Can we just do `Object.prototype.hasOwnProperty.call.bind(Object.prototype.hasOwnProperty)`?

Reply:
No, because that will result in a function equal to:
```js
function (arg0) {
	return Object.prototype.hasOwnProperty.apply(
		Object.prototype.hasOwnProperty,
		arguments
	);
}
```

Reply:
Actually, it&nbsp;should&nbsp;be (one&nbsp;less&nbsp;`[[Get]]` to&nbsp;call):
```js
Function.prototype.call.bind(Object.prototype.hasOwnProperty)
```